### DISCUSSION TITLE
Cannot get themes to save between shell restarts

### DISCUSSION THREAD

**Discussion by antgustech** [JanDeDobbeleer/oh-my-posh#1482] [Troubleshoot] [answered]
Hi, I installed oh py posh using winget. I primarily use git bash on windows. I cloned the repo and then I ran the commands like this.

`eval "$(oh-my-posh --init --shell bash --config C:/projects/oh-my-posh/themes/jandedobbeleer.omp.json)"`

This works, the theme is loaded correctly! However,  I am having some trouble getting it to save it. If I restart the shell, the theme is gone again. I checked my home folder with
`
echo $HOME`

and I do have an empty `.bashrc` file there. Running` . ~/.bashrc` does nothing it seems.

Not sure what I am missing here.

**Comment by JanDeDobbeleer** [comment, marked as answer]
If I understand correctly you're running that command directly in bash whereas you should add that to the `.bashrc` file. That way it will be run when starting bash.

**Reply by antgustech** [reply]
God..yes, I missed that part, thank you. 💯
